### PR TITLE
feat(actions): package and deliver qt6 tarballs

### PR DESCRIPTION
**Pull request by DioEgizio**

### CODE REVIEW

**Comment by txtsd** [comment]
Not back home yet. I'll take a look at it later today or tomorrow.

**Review by txtsd — APPROVED** [review]

**Comment by txtsd** [comment]
@DioEgizio Just noticed, why aren't your commits verified?
Do the thing: <https://docs.github.com/en/authentication/managing-commit-signature-verification>

**Comment by DioEgizio** [comment]
lazyness :O

**Review by flowln — APPROVED** [review]

**Comment by Scrumplex** [comment]
Does this even work? I know that we had issues with Qt 5 builds that used `jurplel/install-qt-action`

**Comment by DioEgizio** [comment]
Seems to work, I asked people if it worked

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -357,22 +356,36 @@ jobs:
...
+        uses: actions/upload-artifact@v3
+        with:
+          name: PolyMC-${{ runner.os }}-qt6-${{ env.VERSION }}-${{ inputs.build_type }}
+          path: PolyMC.tar.gz
+
+      - name: Upload binary tarball (Linux, portable, Qt 6)
+        if: runner.os == 'Linux' && matrix.qt_ver != 5
+        uses: actions/upload-artifact@v3
+        with:
+          name: PolyMC-${{ runner.os }}-Portable-qt6-${{ env.VERSION }}-${{ inputs.build_type }}
```
Comment (Scrumplex) [inline .github/workflows/build.yml]:
```suggestion
          name: PolyMC-${{ runner.os }}-Qt6-Portable-${{ env.VERSION }}-${{ inputs.build_type }}
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -357,22 +356,36 @@ jobs:
...
         uses: actions/upload-artifact@v3
         with:
           name: PolyMC-${{ runner.os }}-Portable-${{ env.VERSION }}-${{ inputs.build_type }}
           path: PolyMC-portable.tar.gz
 
+      - name: Upload binary tarball (Linux, Qt 6)
+        if: runner.os == 'Linux' && matrix.qt_ver !=5
+        uses: actions/upload-artifact@v3
+        with:
+          name: PolyMC-${{ runner.os }}-qt6-${{ env.VERSION }}-${{ inputs.build_type }}
```
Comment (Scrumplex) [inline .github/workflows/build.yml]:
```suggestion
          name: PolyMC-${{ runner.os }}-Qt6-${{ env.VERSION }}-${{ inputs.build_type }}
```

File: .github/workflows/trigger_release.yml [hunk 3]
```diff
@@ -35,6 +35,8 @@ jobs:
       - name: Package artifacts properly
         run: |
           mv ${{ github.workspace }}/PolyMC-source PolyMC-${{ env.VERSION }}
+          mv PolyMC-Linux-Portable-qt6*/PolyMC-portable.tar.gz PolyMC-Linux-Portable-qt6-${{ env.VERSION }}.tar.gz
+          mv PolyMC-Linux-qt6*/PolyMC.tar.gz PolyMC-Linux-qt6-${{ env.VERSION }}.tar.gz
```
Comment (Scrumplex) [inline .github/workflows/trigger_release.yml]:
```suggestion
          mv PolyMC-Linux-Qt6-Portable*/PolyMC-portable.tar.gz PolyMC-Linux-Qt6-Portable-${{ env.VERSION }}.tar.gz
          mv PolyMC-Linux-Qt6*/PolyMC.tar.gz PolyMC-Linux-Qt6-${{ env.VERSION }}.tar.gz          
```

File: .github/workflows/trigger_release.yml [hunk 4]
```diff
@@ -70,6 +72,8 @@ jobs:
             PolyMC-Linux-Portable-${{ env.VERSION }}.tar.gz
             PolyMC-Linux-${{ env.VERSION }}-x86_64.AppImage
             PolyMC-Windows-Legacy-${{ env.VERSION }}.zip
+            PolyMC-Linux-qt6-${{ env.VERSION }}.tar.gz
+            PolyMC-Linux-Portable-qt6-${{ env.VERSION }}.tar.gz
```
Comment (Scrumplex) [inline .github/workflows/trigger_release.yml]:
```suggestion
            PolyMC-Linux-Qt6-${{ env.VERSION }}.tar.gz
            PolyMC-Linux-Qt6-Portable-${{ env.VERSION }}.tar.gz
```

**Review by Scrumplex — APPROVED** [review]

**Comment by DioEgizio** [comment]
3 approvals 👀

**Pull request merged**